### PR TITLE
Add os.{read,write,remove,unlink}

### PR DESCRIPTION
**Pull request by palaviv**
The test should be changed to create the file in some temp folder once we add `tempfile`. In case anyone has a simpler idea that will be more safe please suggest.

### CODE REVIEW

File: vm/src/stdlib/os.rs [hunk 1]
```diff
@@ -113,6 +113,25 @@ fn os_error(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     Err(vm.new_os_error(msg))
 }
 
+fn os_read(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
```
Comment (adrian17) [inline vm/src/stdlib/os.rs]: These functions probably would be easy to write with new args style, something like:

```
fn os_read(fd: PyIntRef, n: PyIntRef, vm: &VirtualMachine) -> PyResult
```

Comment (palaviv) [reply vm/src/stdlib/os.rs]: Done.

**Comment by windelbouwman** [comment]
Looks good to me! Testing seems tricky, we can also skip testing if we are going to call file deletion.

**Comment by windelbouwman** [comment]
Maybe I would suggest to not test file deletion and creation, since it depends heavily on the directory where tests are run. It's pretty tricky. Or we should define a seperate test set aside from the snippets which does this kind of functional tests. Maybe we could use BDD or robotframework to test those behaviors with files and network connections.

**Comment by palaviv** [comment]
My plan is to add `mkdir` and `rmdir` and move the tests into a temp directory. I am not sure if we need a new framework but moving them aside and test them in a separate travis might be good.

**Comment by adrian17** [comment]
> I am not sure if we need a new framework but moving them aside and test them in a separate travis might be good.

IMO with temporary directories (as in, system's temporary directory, not `mkdir ./tmp`) that would be good enough in normal tests.

**Pull request merged**